### PR TITLE
Implemented the 'Get Started' page for UD

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Implemented the 'Get Started' page for UD.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15520

Screenshot from 2020-01-13 13-12-12
out-3

### CODE REVIEW

**Review by akurinnoy — APPROVED** [review]

File: dashboard/src/app/dashboard/dashboard-config.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {DashboardLastWorkspacesController} from './last-workspaces/last-workspac
 import {DashboardLastWorkspaces} from './last-workspaces/last-workspaces.directive';
 import {DashboardPanel} from './dashboard-panel/dashboard-panel.directive';
 import {CheWorkspace} from '../../components/api/workspace/che-workspace.factory';
+import  {MENU_ITEM}  from '../navbar/navbar.controller';
```
Comment (sleshchenko) [inline dashboard/src/app/dashboard/dashboard-config.ts]: redundant space seems to appear here

File: dashboard/src/app/demo-components/demo-components.html [hunk 2]
```diff
@@ -306,27 +306,26 @@ <h6>This is h6
...
             <che-list-item ng-repeat="task in demoComponentsController.listItemsTasks">
-              <div layout="row" flex="100" style="height: 24px; margin: 5px">
+              <div layout="row" flex="100" style="height: 35px; margin: 5px">
                 <div flex="70">{{task.name}}</div>
                 <che-list-item-checked flex="30" ng-model="task.done"></che-list-item-checked>
               </div>
             </che-list-item>
           </che-list>
-        </che-html-source>
+          </textarea>
```
Comment (sleshchenko) [inline dashboard/src/app/demo-components/demo-components.html]: Seems it's not aligned with open tag.

**Review by sleshchenko — APPROVED** [review]
LGTM

File: dashboard/src/app/demo-components/demo-components.html (hunk 2 above)
Comment (olexii4) [reply dashboard/src/app/demo-components/demo-components.html]: Ok. Thanks.
Done

File: dashboard/src/app/dashboard/dashboard-config.ts (hunk 1 above)
Comment (olexii4) [reply dashboard/src/app/dashboard/dashboard-config.ts]: Done

**Comment by beaumorley** [comment]
This looks great. Once the logos are replaced with the correct ones, I approve from UX perspective. Thanks!

**Comment by SkorikSergey** [comment]
crw-ci-test

**Review by SkorikSergey — APPROVED** [review]

**Comment by slemeur** [comment]
Well done guys!

**Pull request merged**